.golangci.yml

```diff
@@ -28,4 +28,5 @@ run:
   deadline: 4m
   skip-files:
     - "zz_generated.*\\.go$"
+    - ".*conversion.*\\.go$"
```

Comment:
I'd prefer if we didn't avoid all linting on these file, since it does contain non-generated code. I'm wondering if we can use an exclude string similar to line 26 of `.golangci.yml` to handle the common error(s) in a way that doesn't affect other files and/or stick with using the comment based linting and exclude the specific linter that is an issue (i.e `//nolint:prealloc`) like we do in other cases.

Reply:
Yeah I agree, we should probably use the other block. We're doing the same upstream in CAPI, and maybe when we find an actual solution we should suggest it to all providers as well.

Do you want me to revert this change for this PR?

Reply:
Yeah, I think revert and converge on a better solution down the line is the best path forward.